docs/security/measures.md

```diff
@@ -161,3 +161,48 @@ add_header X-XSS-Protection "1; mode=block";
 ```
 
 [HSTS]: https://www.nginx.com/blog/http-strict-transport-security-hsts-and-nginx/
+
+### Add Fail2Ban filter to increase protection further
+
+If you are running fail2ban to protect your system, you can also add a filter and jail for authelia to reduce load on the application / web server from repeated hacking attempts.
+
+If you are using docker, the authelia log file location has to be mounted from the host system to the container for fail2ban to work. Otherwise fail2ban is unable to access it.
```

Comment:
```suggestion
If you are using docker, the Authelia log file location has to be mounted from the host system to the container for fail2ban to work. Otherwise fail2ban is unable to access it.
```